src/main/java/pl/javastart/task/VehicleWarehouse.java

```diff
@@ -0,0 +1,33 @@
+package pl.javastart.task;
+
+public class VehicleWarehouse {
+    private Vehicle[] vehicles = new Vehicle[5];
```

Comment:
Jeżeli w tablicy masz tylko samochody i ciężarówki, to możesz użyć typu `Car[]`, a nie `Vehicle[]`, dzięki temu nie będziesz musiał dodawać metod związanych z klimatyzacją do Vehicle.

Ogólna zasada jest taka, że "używaj najogólniejszego typu referencji, który nie powoduje utraty funkcjonalności". Tutaj typ `Vehicle` jest trochę zbyt ogólny, przez co musiałeś sztucznie dodawać metody typu `acTurnOn` do Vehicle